src/main/app/src/components/koulutus/TulevaJarjestajaList.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Typography, Grid, Container, makeStyles } from '@material-ui/core';
+import Spacer from '../common/Spacer';
+import { Localizer as l } from '../../tools/Utils';
+import OppilaitosCard from './OppilaitosCard';
+import { useTranslation } from 'react-i18next';
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    marginTop: '100px',
+  },
+});
+
+const localizeArrayToString = (toLocalizeArray) => {
+  return toLocalizeArray
+    .map((item) => l.localize(item.nimi))
```

Comment:
```suggestion
    .map((item) => l.localize(item))
```
Muistaakseni tässä `nimi` erikseen ei tarvitse kirjoittaa, asetetaan utilsin sisällä